lib/virtmanager.pm

```diff
@@ -15,9 +15,11 @@ sub launch_virtmanager() {
     type_string "virt-manager", 50;
     send_key "ret";
     wait_idle;
-    type_password;
-    send_key "ret";
-    wait_idle;
+    if (check_var("FLAVOR", "Server-DVD")) {
```

Comment:
the whole virtmanager test suite is like that ;(

But I'm afraid this is just moving the problem to the next one who actually needs to test this on a server. Because the provided test results are only for desktop, so all we know: we don't need this code on SP2